tensorrt_llm/_torch/models/modeling_deepseekv3.py

```diff
@@ -182,7 +182,8 @@ def __init__(
         topk_group: int,
         routed_scaling_factor: float,
         dtype: Optional[torch.dtype] = None,
-        is_thop: bool = True,
+        fuse_routing_kernel: bool = True,
+        apply_routing: bool = False,
```

Comment:
In which case will `apply_routing` will `True`?  It seems tha `apply_routing` is always `False` in this `modeling_deepseekv3.py`.

Reply:
It's more like a tag to tell the user that the routing is skipped in this module.